tm2/pkg/crypto/keys/client/maketx.go

```diff
@@ -180,7 +180,8 @@ func ExecSignAndBroadcast(
 	io.Println(string(bres.DeliverTx.Data))
 	io.Println("OK!")
 	io.Println("GAS WANTED:", bres.DeliverTx.GasWanted)
-	io.Println("GAS USED:  ", bres.DeliverTx.GasUsed)
+	io.Println("GAS USED:", bres.DeliverTx.GasUsed)
+	io.Println("EVENTS:", string(bres.DeliverTx.EncodeEvents()))
 
```

Comment:
I believe this is a suitable location to display block height and timestamp (new line) instead of including this information in every event.

Reply:
For now, I've made it so that only the `Height` is output. This is because the timestamp is not currently stored separately in the `ResultBroadcastTxCommit` type, and adding it seems like it would be a significant undertaking.

Reply:
I believe the timestamp is stored in a subobject. Can someone verify? @zivkovicmilos

Reply:
cc @gnolang/tech-staff

Reply:
@moul 

Apologies for the late reply on this 

There is no way to fetch the transaction timestamp, unless you directly tie it somehow to a block (we don't store the tx timestamp in any tx resulting type, or for any `checkTx` / `deliverTx` call).

The block header has a `Time` field, and if you know which block the tx belongs to you can figure out when it was committed (ie. `block timestamp == tx timestamp`)

Reply:
Resolving because this convo is outdated